src/components/ChatLog.jsx

```diff
@@ -0,0 +1,42 @@
+import './ChatLog.css';
+import ChatEntry from './ChatEntry.jsx';
+import PropTypes from 'prop-types';
+
+const ChatLog = (props) => {
+  const chatMessages = props.entries.map((message) => {
+    return (
+      <li key={message.id}>
+        <ChatEntry id={message.id} sender={message.sender} body={message.body}
+          timeStamp={message.timeStamp} liked={message.liked}
+          onLikeToggle={props.onMessageLikeToggle} colorChoice={props.colorChoice} twoPeople={props.twoPeople}>
+        </ChatEntry>
+      </li>
```

Comment:
There are extra layers to the HTML that aren't necessary. Right now `ChatEntry` is wrapped in an `li` at the `ChatLog` layer and gets wrapped in another `div` at the `ChatEntry` layer. 

We could remove the `li` tags in `ChatLog` and replace the outermost `div` in `ChatEntry` with `li` so that we do not use a `div` at all, factoring out elements that are not semantic and removing a layer of wrapping!